_extras/Recipe.md

```diff
@@ -0,0 +1,76 @@
+* See below with regard to timing challenges. If you are committed to covering the same content that you would in an in-person workshop, you may wish to extend your planned time.
+### Planning meetings
+* Plan at **least 2 meetings** before the workshop. 
+    * **Meeting 1 (well in advance)**: Establish workshop roles, who teaches what, delegate documentation & communications setup (below), plan asynchronous preparation.
+    * **Meeting 2 (closer to the date)**: Review workshop mechanics, backup planning, learner communications, assessment, and feedback review. Agree on content to cut when time runs short. Test video conferencing features.
+    
+For more detailed preparation goals, see our [checklist for online workshop preparation](https://carpentries.org/online-workshop-recommendations/#planning-your-workshop).
+### Back-channel communications
+* **Create a channel** or private conversation thread on The Carpentries Slack workspace (or other preferred tool). 
+* **Use your communication channel** to communicate about preparations. This will allow everyone on your instructional team to become familiar with the tool as well as improve your preparations.
+### Documentation
+* **Create an Etherpad** or other collaborative document, and add all vital information to be included in your workshop introduction for learners who arrive late. This should include:
+    * Workshop schedule (including all breaks)
+    * Connection instructions
+    * What to do in the event of a connection failure
+    * How to ask questions
+    * Other instructions related to workshop mechanics, including recommended window arrangement
+    * Links to The Carpentries Code of Conduct as well as your workshop website
+* Add a link to your Etherpad to your **workshop website**.
+* For a self-organised workshop, be sure to submit your [workshop request form](https://amy.carpentries.org/forms/self-organised/).
+* **Email your learners** with everything they need to know in advance of the workshop. 
+    * Our [template for in-person trainings](https://docs.carpentries.org/topic_folders/workshop_administration/email_templates.html#email-learners-before-workshop) may be useful. Much of this info may already be on your Etherpad, so you can be selective about information that needs duplicating.
+### Preparing for Adjustments
+* **Plan to cut content.** Online workshops often run more slowly than they do in-person; sometimes much more slowly. Decisions to cut back should be intentional and collaborative.
+    * Remember that skipping steps in a workflow can create problems with dependencies later on.
+* **Agree on your priorities.** Discussing key objectives and broader goals with your instructional team in advance will help everyone navigate difficult trade-offs when time becomes an issue during a workshop.
+* **Have a backup** for all roles and documents. Technical failures come in all shapes and sizes, and life can still happen, too!
```

Comment:
The scaffolding might also help as a back up to installation issues -https://carpentries.org/blog/2020/04/scaffolds/